governance/polls/Inclusion Poll - MIP0 Amendments (MIP4c2-SP14) - May 10, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Inclusion Poll for MIP0 Amendments (MIP4c2-SP14) - May 10, 2021
+summary: Signal your support or opposition for this proposal being included in this month's governance cycle. 
+discussion_link: https://forum.makerdao.com/t/mip4c2-sp14-mip0-amendments/7260
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Inclusion Poll for MIP0 Amendments (MIP4c2-SP14) - May 10, 2021
+
+The Governance Facilitators have placed an inclusion poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP3](https://mips.makerdao.com/mips/details/60626de7e65b747f996b3d4e). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 10 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
```

Comment:
Still have an open PR for adding descriptions for the MIPs. It was mainly for the Amendments, and the other titles are clear enough but here I think we should add it in. I was thinking:
MIP0 Amendments (MIP4c2-SP14): Clarifies and beautifies MIP0 to facilitate Governance participation.